src/expandable-section/internal.tsx

```diff
@@ -109,7 +109,7 @@ export default function InternalExpandableSection({
         <ExpandableSectionHeader
           id={triggerControlId}
           descriptionId={descriptionId}
-          className={clsx(styles.header, styles[`header-${baseVariant}`])}
+          className={clsx(styles.header, styles[`header-${baseVariant}`], 'awsui-screenreader-text')}
```

Comment:
The `awsui-screenreader-text` class name is used to locate the part of the expandable section to get extracted for screenreader announcement. It it used in the series-detail-text util. The same approach is to be reused in the chart-components, where we can't use the hashed internal class name.

Reply:
Can we solve this alternatively by extracting a selector with this class to a scss file to be added to the `internal/do-not-use` directory?

This is an approach we are already using, and we would avoid customers using this class name directly.